tests/DsnTest.php

```diff
@@ -171,6 +171,16 @@ public function createFromStringThrowsExceptionIfValueIsInvalidDataProvider(): \
             'tcp://public:secret@example.com/1',
             'The scheme of the "tcp://public:secret@example.com/1" DSN must be either "http" or "https".',
         ];
+
+        yield 'valid project id' => [
```

Comment:
As far as I know you cannot have two test cases with the same name/description. Please change this description to `invalid project ID (char instead of number)`